packages/library-sync/src/commonMain/kotlin/io/realm/kotlin/mongodb/sync/SyncConfiguration.kt

```diff
@@ -53,7 +56,7 @@ public enum class SyncMode {
      * which is a property that must be set on all objects. Server objects that
      * match a given _partition value_ are then synchronized to the device.
      *
-     * @see https://www.mongodb.com/docs/atlas/app-services/sync/data-access-patterns/partitions/
+     * **See:** [Partitions](https://www.mongodb.com/docs/atlas/app-services/sync/data-access-patterns/partitions/)
```

Comment:
Nope, I just noticed it yesterday. Pretty much all our references to `@see` are wrong 🙈 It can only be used to browse the API, not external links: https://kotlinlang.org/docs/kotlin-doc.html#see-identifier. I found some people using this as an alternative: https://stackoverflow.com/questions/53161281/how-to-represent-hyperlink-in-kdoc